docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md

```diff
@@ -11,7 +11,8 @@ This quickstart is designed for linux based systems such as Linux distros and Ma
 1. **Docker** - Ensure that your system has docker engine installed and running. For installation, please refer to [docker engine installation instruction](https://docs.docker.com/engine/install/).
    * The recommended way is to install docker desktop for your system. If using WSL with Windows, Enable docker for you WSL distro. Please refer to [docker desktop installation](https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers#install-docker-desktop) for more details.
 2. **Docker Compose** - Ensure docker compose is enabled along with docker engine. Please refer to [docker compose installation instruction](https://docs.docker.com/compose/install/).
-3. **Loophole** - Loophole is required to expose your local system to the services on web by tunneling the external calls to your local system. Please refer to [Loophole Quickstart Guide](https://loophole.cloud/download/) for installation.
+3. **Loophole** - Loophole is required to expose your local system to the services on the web by tunneling the external calls to your local system. Please refer to the [Loophole Quickstart Guide](https://loophole.cloud/download/) for installation.
```

Comment:
Can we make both loophole and ngrok as subpoints just to tell them apart as either/or and give a heading to point 3 (eg: tunnel)?